packages/android-sdk-build-tools/PKGBUILD

```diff
@@ -14,23 +14,23 @@ license=('custom:unknown')
 depends_i686=('gcc-libs' 'zlib')
 depends_x86_64=('lib32-gcc-libs' 'lib32-zlib')
 _sdk=android-sdk
+
 source=("https://dl-ssl.google.com/android/repository/build-tools_$pkgver-linux.zip")
-sha512sums=('61d17c3e21f51e6c7589b688eaa87e93862f189ec33e0df64413fae510d6602178348ef9833cb00f5b1071cf5baf306b4a25d9b9f8520a3e722ad0f9451e0942')
-_android=android-8.1.0
+sha512sums=('845e527b3ef809d5e32bdce0023401ae80c545504f152d7c05ccef64e4006def6cbda07a295a66518356e992abbbe8f30522c5e63d6eb12a1651c614643602d8')
+_android=android-10
 options=('!strip')
 
 package() {
-  cd $pkgdir
-
-  install -Dm 644 "$srcdir/$_android/NOTICE.txt" \
-    "usr/share/licenses/$pkgname/NOTICE.txt"
 
-  ver=$(cat "$srcdir/$_android/source.properties" |
-  grep ^Pkg.Revision=|sed 's/Pkg.Revision=\([0-9.]*\).*/\1/')
-
-  install -dm 755 opt/$_sdk/build-tools/$ver
+  cd "$pkgdir"
+  install -Dm644 "${srcdir}/$_android/NOTICE.txt" usr/share/licenses/$pkgname/NOTICE.txt
+  # mkdir -p opt etc/profile.d
+  # echo 'export PATH=$PATH:/opt/android-sdk/build-tools/'"$_ver/" > etc/profile.d/${pkgname}.sh
+  # echo 'setenv PATH ${PATH}:/opt/android-sdk/build-tools/'"$_ver/" > etc/profile.d/${pkgname}.csh
+  # chmod 755 etc/profile.d/${pkgname}.{csh,sh}
+  ver=$(cat "${srcdir}/$_android/source.properties" |grep ^Pkg.Revision=|sed 's/Pkg.Revision=\([0-9.]*\).*/\1/')
+  mkdir -p opt/$_sdk/build-tools/$ver
```

Comment:
You missed `$pkgdir` here, the next command will not work. Also use `install -dm` instead of mkdir.